### DISCUSSION TITLE
Building from source, missing node_modules

### DISCUSSION THREAD

**Discussion by DewitteRuben** [gitpod-io/openvscode-server#307] [Q&A]
Hi,

I followed the docs and was successfully able to build from source using `yarn gulp vscode-reh-web-linux-x64-min`.

However, when I try to run `./bin/openvscode-server` it would complain some modules are missing. (`Error: Cannot find module 'minimist'` in this case)

So I compared my manual release to one that is released by you guys on Github, with that I can see the node_modules are missing in my case.

Where am I supposed to get these node_modules from? When running the build command in older versions of this repo (such as v1.63.2), the node_modules are automatically included in the build output.

I tried copying over the node_modules from the openvscode-server source that were created on `yarn install` but that didn't seem to work.

Is there some other step that I'm missing to have this working?

**Comment by jeanp413** [comment]
Hi @DewitteRuben, there was a [change upstream](https://github.com/microsoft/vscode/commit/b7712c07fac598aa627b6d13eb488ce02023c678) recently so `node_modules` for remote folder are not installed automatically, you could try running this commands to get all dependencies installed, let me know if that works for you
```
yarn
yarn --cwd remote
yarn --cwd remote/web 
```
